src/test/users-tests.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../app';
+
+dotenv.config();
+
+
+chai.use(chaiHttp);
+const { expect } = chai;
+chai.should();
```

Comment:
You have two libraries achieving the same thing. could you please considering using one